app/views/stats/index.html.slim

```diff
@@ -34,4 +34,12 @@ hr
                 span.visually-hidden
                   = " #{ report.full_name }"
 
-
+        - if current_user.admin?
```

Comment:
I think we should maybe move this whole thing to the Admin dashboard then we shouldn't need to have this logic in the view as the policy of the dashboard page would control who can see this report.